src/test/java/at/ac/tuwien/kr/alpha/solver/SolverTests.java

```diff
@@ -76,8 +71,8 @@ public int compareTo(Thingy o) {
 		}
 	}
 
-	@Test
-	public void testObjectProgram() throws IOException {
+	@RegressionTest
+	public void testObjectProgram(RegressionTestConfig cfg) throws IOException {
```

Comment:
Thanks! These seem to be leftovers from way back since they weren't even required before the JUnit upgrade. I removed the `throws` statements in question.